<launch>
  <arg name="n_obstacles" default="8"/> 
  <arg name="publish_at_lookahead" default="false"/> <!-- true, false -->
  <arg name="lookahead_distance" default="5.0"/> 
  <arg name="rnd_seed" default="84"/>  <!-- int -->
  <arg name="obstacle_width" default="0.2"/>
  <arg name="obstacle_length" default="0.3"/>
  <arg name="obstacle_max_d_from_traj" default="1.0"/>

<node name="random_obstacle_publisher" pkg="random_obstacle_publisher" type="random_obstacle_publisher.py" 
       output="screen">
  <param name="n_obstacles" type="int" value="$(arg n_obstacles)" />
  <param name="publish_at_lookahead" type="bool" value="$(arg publish_at_lookahead)" />
  <param name="lookahead_distance" type="double" value="$(arg lookahead_distance)" />
  <param name="obstacle_width" type="double" value="$(arg obstacle_width)" />
  <param name="obstacle_length" type="double" value="$(arg obstacle_length)" />
  <param name="obstacle_max_d_from_traj" type="double" value="$(arg obstacle_max_d_from_traj)" />
  <param name="rnd_seed" type="int" value="$(arg rnd_seed)" />
</node>

</launch>